lib/trubl/api/search.rb

```diff
@@ -49,7 +49,8 @@ def search(type, query, per_page=nil, page=nil, filter_options={})
       end
 
       def filter_options_whitelist
```

Comment:
I think in reality the server should be responsible for deciding what parameters to accept rather than the client. Seems rather trusting...

Not sure what the impact would be if you deleted it altogether though...